### PR TITLE
Darreldonald/hideimage

### PR DESCRIPTION
**Pull request by DarrelDonald**
fix success panel dark theme after publishing/saving detector

add capability to hide an image when showing the code in docs

corrected namespace

### CODE REVIEW

**Comment by DarrelDonald** [comment]
image

File: AngularApp/projects/applens/src/app/modules/dashboard/applens-doc-section/applens-doc-section.component.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { getConfigFileParsingDiagnostics } from 'typescript';
 import { ApplensDiagnosticService } from '../services/applens-diagnostic.service';
 import * as momentNs from 'moment';
 import { GenericThemeService } from 'projects/diagnostic-data/src/lib/services/generic-theme.service';
+import { $ } from 'protractor';
```
Comment (XiaoxuHU) [inline AngularApp/projects/applens/src/app/modules/dashboard/applens-doc-section/applens-doc-section.component.ts:15]: If not need, you can remove this import

File: AngularApp/projects/applens/src/app/modules/dashboard/applens-doc-section/applens-doc-section.component.ts [hunk 2]
```diff
@@ -207,6 +209,8 @@ export class ApplensDocSectionComponent implements OnInit {
   toggleDisplayCode(){
     this.codeHidden = !this.codeHidden;
     this.displayCodeButtonText = this.displayCodeButtonText === "Show Code" ? "Hide Code" : "Show Code";
+    if(this.image != "")
+      document.getElementById(this.image).style.display = this.codeHidden ? 'unset' : 'none';
```
Comment (XiaoxuHU) [inline AngularApp/projects/applens/src/app/modules/dashboard/applens-doc-section/applens-doc-section.component.ts:213]: Just wondering, why not using *ngIf for hide and show the image ?

File: AngularApp/projects/applens/src/app/shared/components/login/login.component.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ export class LoginComponent implements OnInit {
 
   interval: any;
 
-  browser: IUAParser.IBrowser;
+  browser: UAParser.IBrowser;
```
Comment (XiaoxuHU) [inline AngularApp/projects/applens/src/app/shared/components/login/login.component.ts:22]: Revert this change, it caused your [build pipeline](https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_build/results?buildId=11254&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=9c939e41-62c2-5605-5e05-fc3554afc9f5) failed

File: AngularApp/projects/applens/src/app/modules/dashboard/applens-doc-section/applens-doc-section.component.ts (hunk 2 above)
Comment (DarrelDonald) [reply AngularApp/projects/applens/src/app/modules/dashboard/applens-doc-section/applens-doc-section.component.ts:213]: this is for an image added with markdown

**Review by XiaoxuHU — APPROVED** [review]
